elastio-lambda-stream-backup/readme.md

```diff
@@ -0,0 +1,68 @@
+Before starting, please check do you have aws-cliv2, and docker cli installed.
+
+1. From AWS console, select IAM, select Roles then create IAM role with "AdministratorAccess".  
```

Comment:
Check with @sokyrko , soon we will have (or maybe already have in `master`) Elastio-specific IAM policies that will let you create roles that have enough permissions to do backups like stream backup without granting full admin rights.

Reply:
It's already in master.

Reply:
"AdministratorAccess" was changed to "ElastioFullAdmin"